tests/pytest/test.py

```diff
@@ -1114,6 +1115,46 @@ def testCopyCommand(env):
     r.assertEqual(json.loads(res), [new_values])
 
 
+def nest_object(depth, max_name_len, leaf_key, leaf_val):
```

Comment:
Why not build it as python dict and then decode it as json?

Reply:
Initially tried with python dict but It was throwing some exception

Reply:
So lets understand the exception and fix them...

Reply:
Fixed